neo/Prompt/Commands/Invoke.py

```diff
@@ -210,7 +210,7 @@ def TestInvokeContract(wallet, args, withdrawal_tx=None, parse_params=True, from
     return None, None, None, None
 
 
-def test_invoke(script, wallet, outputs, withdrawal_tx=None, from_addr=None):
+def test_invoke(script, wallet, outputs, withdrawal_tx=None, from_addr=None, net_fee_override=None):
```

Comment:
Would be easier to do this:
`def test_invoke(script, wallet, outputs, withdrawal_tx=None, from_addr=None, net_fee_override=Fixed8.FromDecimal(.001)):`

Then you can get rid of the conditional below.

Reply:
Also may make sense to just use the kwarg name `net_fee`

Reply:
So, I actually implemented that way initially, but with 4 different methods having this arg (and thus specifying the default value), I opted for the `net_fee_override` approach instead so that the default of `.001` is limited to just the two places it's used. I'm happy to change it if you guys think that's preferable.

I might suggest instead that the value is just the float value, though, so that callers don't have to deal with calling `Fixed8.FromDecimal()`; it'll be easier to just pass in the float value directly and let the method handle the type conversion.

Thoughts?

Reply:
Actually, I'll just make it use a constant. I think that'll make the most sense. Update incoming shortly...

Reply:
I decided to go with the name `min_fee` for the kwarg to disambiguate it from the existing `net_fee` variable used in `test_invoke`. I also opted for your suggestion of defining the kwarg as `Fixed8` for consistency with other `fee` args.

Let me know if you think anything else needs tweaking! Thanks for the feedback 👍